Admin/AbstractAdmin.php

```diff
@@ -2017,7 +2017,7 @@ public function buildBreadcrumbs($action, MenuItemInterface $menu = null)
             E_USER_DEPRECATED
         );
         if ($this->breadcrumbsBuilder === null) {
-            $this->breadcrumbsBuilder = new BreadcrumbsBuilder();
+            $this->breadcrumbsBuilder = new BreadcrumbsBuilder($this->getConfigurationPool()->getContainer()->getParameter('sonata.admin.configuration.breadcrumbs'));
```

Comment:
Might be needed by people who overrode the template that contains the breadcrumbs, and still call the breadcrumbs builder the old way. So yeah, it will work for most people if we remove this.